lib/ruy/outcome.rb

```diff
@@ -1,24 +1,39 @@
 module Ruy
+  # Represents a possible result (outcome) of evaluating a rule set
+  #
+  # Additionally, an outcome can be composed of rules. In that case, the outcome will evaluate to
```

Comment:
Additional lines in description need to be indented in order for YARD to understand they belong to the same comment block.